common/app/model/dotcomrendering/pageElements/PageElement.scala

```diff
@@ -236,18 +236,12 @@ object GenericAtomBlockElement {
   implicit val GenericAtomBlockElementWrites: Writes[GenericAtomBlockElement] = Json.writes[GenericAtomBlockElement]
 }
 
-case class GuideAtomBlockElementItem(title: Option[String], body: String)
```

Comment:
Not sure how this is related to the prebid permutive change?

Reply:
What happened there, I don't know!